source/atomic_counter.cpp

```diff
@@ -57,19 +53,21 @@ AtomicCounter::~AtomicCounter() {
 }
 
 AtomicCounter::WaitingFiberBundle::WaitingFiberBundle()
-		: InUse(true),
```

Comment:
I'm not sure if we can do this in clang format, but if we can keep the double indent of initializer lists, that would be awesome. It helps differentiate the initializer list from the constructor body code. If not, I can live with it.

One other thing I noticed though, is that it seems to be indenting the initializer lists with spaces. It should be tabs.

Reply:
Yeah that can be done. However, it considers it "spacing" as opposed to "indentation" hence the use of spaces.

Reply:
Fixed.